cityehr-quick-start-guide/pom.xml

```diff
@@ -25,6 +25,13 @@
         <fo.generated-resources>${project.build.directory}/generated-resources/fo</fo.generated-resources>
         <html.generated-resources>${project.build.directory}/generated-resources/html</html.generated-resources>
         <website.output.folder>${project.build.directory}/website</website.output.folder>
+
+        <petal.base-url>https://petal.evolvedbinary.com/</petal.base-url>
```

Comment:
Could we rename `petal-base-url` to `petal-api-url` please? (also in XSLT, and CI scripts)

Reply:
Fixed with a12f8e5